app/components/forms/order-form.js

```diff
@@ -0,0 +1,150 @@
+import Ember from 'ember';
+import FormMixin from 'open-event-frontend/mixins/form';
+import { countries } from 'open-event-frontend/utils/dictionary/demography';
+
+const { Component, computed } = Ember;
+
+export default Component.extend(FormMixin, {
+  holders: [1, 2, 3],
+  fillDetails() {
```

Comment:
@niranjan94 I have tried to write method to fill the details but it's not working. Can you please help me out? Thanks